### PR TITLE
Introduce maintenance section

### PR DESCRIPTION
**Pull request by osmantoplica-trackunit**

### CODE REVIEW

File: sdk-docs/maintaining-your-app.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+### Upgrading SDK Packages
+We will periodically release updates for our SDK libraries. It is crucial to keep your applications using the latest versions to maintain compatibility as our platform evolves. You may receive a warning in the terminal when publishing your app if your SDK is outdated. While you can initially ignore this warning, outdated SDK versions may eventually result in your app not being approved or even removed from the platform.
+
+### Updating Your Nx Workspace
+The SDKs rely on specific Nx and Node.js versions. Ensure that your application complies with the required versions at all times. Do *not* update your Nx workspace immediately when a new version is released. Instead, only upgrade when our SDKs depend on a newer version.
```
Comment (mmt-trackunit) [inline sdk-docs/maintaining-your-app.md:18]: SDK in singular form

Comment (mmt-trackunit) [inline sdk-docs/maintaining-your-app.md:18]: > immediately when a new **NX** version is released

Comment (mmt-trackunit) [inline sdk-docs/maintaining-your-app.md:18]: It's difficult to know when our packages depend on a new NX version. But the prerequisites page will be updated and warnings will tell.

File: sdk-docs/maintaining-your-app.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Migrating an existing Nx workspace to a new version may not be a straightforward process. The migration may require manual intervention, such as deleting, moving, renaming, or creating certain files. The migration process depends on the Nx team's implementation. Sometimes they provide a tool that automatically migrates older versions to newer ones, while other times you may need to take additional steps. Always follow the provided guidelines and best practices for a smooth migration experience.
```
Comment (mmt-trackunit) [inline sdk-docs/maintaining-your-app.md:20]: https://nx.dev/packages/nx/documents/migrate

**Review by mmt-trackunit — CHANGES_REQUESTED** [review]

**Comment by osmantoplica-trackunit** [comment]
@mmt-trackunit Let me know if it looks good now

File: sdk-docs/maintaining-your-app.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Updating Your Nx Workspace
+The SDK relies on specific Nx and Node.js versions. Ensure that your application complies with the required versions at all times. Do *not* update your Nx workspace immediately when a new Nx version is released. Instead, only upgrade when our SDKs depend on a newer version. If in doubt, refer to the [prerequisites](hhttps://developers.trackunit.com/docs/prerequisites) page.
+It will be updated whenever our packages depend on new Nx and Node versions. Warnings in your terminal will also notify you that your app is using the wrong versions.
+
+[Migrating](https://nx.dev/packages/nx/documents/migrate) an existing Nx workspace to a new version may not be a straightforward process.
+The migration may require manual intervention, such as deleting, moving, renaming, or creating certain files.
+The migration process depends on the Nx team's implementation. Sometimes they provide a tool that automatically migrates older versions
```
Comment (mmt-trackunit) [inline sdk-docs/maintaining-your-app.md:23]: @mta-trackunit Is this correct? Isn't there always a migration script?

Comment (mta-trackunit) [reply sdk-docs/maintaining-your-app.md:23]: its automatically upgrades only - unless you dont follow the standard way of doing things

**Review by mmt-trackunit — APPROVED** [review]

**Pull request merged**